src/modules/auth/auth.module.ts

```diff
@@ -1,8 +1,11 @@
 import { Module } from '@nestjs/common';
 import { AuthService } from './auth.service';
 import { AuthController } from './auth.controller';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from '../user/entities/user.entity';
```

Comment:
In the future, let's make sure that we are not going to inject repositories. Better to import the services from other modules

Reply:
okay, got it